最近整理的生物信息方面的导师，会持续更新，仅供大家参考

知乎地址：生物信息学导师推荐 - Jade的文章 -  https://zhuanlan.zhihu.com/p/441702265

[TOC]

## 一：表观遗传组学:

### Dana-Farber and Harvard Medical School

#### 刘小乐

主要方向：整合分析全基因组ChIP-chip/Seq、核小体定位、组蛋白修饰数据、开发生物信息学算法

主要成果：开发Model-based analysis of ChIP-Seq (MACS)、Cistrome数据库

Homepage: [Liu Lab](https://liulab-dfci.github.io/)

注：已离开学术界，任职于GV20 Oncotherapy，董事会主席）

#### Bradley E. Bernstein

主要方向：Epigenomic Regulation in Development and Cancer

主要成果：The NIH Roadmap Epigenomics Mapping Consortium

Homepage: https://bernstein.dfci.harvard.edu/

#### Nancy Kleckner

主要方向：The Physical Biology of Chromosomes

 主要成果：3C（Capturing Chromosome Conformation，通讯）

Homepage: [Nancy Kleckner](https://projects.iq.harvard.edu/kleckner_lab/home)

### University of Massachusetts Medical School

#### Job Dekker(导师：[Nancy Kleckner](#Nancy Kleckner))

主要方向：Genome folding；Genetic and spatial organization of the extraordinary dinoflagellate chromosomes

主要成果：3C（Capturing Chromosome Conformation，一作）；Hi-C

Homepage: [Job Dekker - Dekker Lab](http://www.dekkerlab.org/job-dekker/)

### 浙江大学

#### 阮一俊

主要方向：3D genome biology

主要成果：ChIA-PET；参与ENCODE计划

Homepage: [Yijun Ruan](https://scholar.google.com/citations?user=pGiG2REAAAAJ&hl=en)

### University of California, Irvine

#### 李蔚（导师：[陈润生](#陈润生)）

主要方向： 染色质修饰与基因调控、DNA 甲基化

主要成果：RSeQC: quality control of RNA-seq experiments

Homepage: [UC Irvine - Faculty Profile System - Wei Li](https://www.faculty.uci.edu/profile.cfm?faculty_id=6687)

### Fred Hutchinson Cancer Research Center

#### Steven Henikoff

主要方向：genomic tools to the study of proteins of the epigenome: histones, transcription factors, nucleosome remodelers, and RNA polymerase II (RNAPII)

主要成果：ChEC-seq；DNase-seq；CUT&Tag；CUT&Run；CUT&Tag2for1；MulTI-Tag

Homepage: https://research.fredhutch.org/henikoff/en.html

### University of California San Diego

#### 任兵

主要方向：非编码序列如何指导基因表达的时空模式，表观遗传机制如何在发育过程中调节这些序列的作用

主要成果：chip-chip；识别和表征增强子的转录控制元件；Paired-seq；Paired-tag

Homepage: [Bing Ren, Ph.D.](http://renlab.sdsc.edu/renlab_website/bing/)

### Massachusetts Institute of Technology(MIT)

#### Manolis Kellis

主要方向：epigenetic modifications to define chromatin states；comparative genomics；annotation of the non-coding genome；Regulatory Networks

主要成果：参与构建ENCODE数据库；ChromHMM: automating chromatin-state discovery and characterization

Homepage: [Manolis Kellis (Kamvysselis)](http://web.mit.edu/manoli/)

### Stanford University

#### William Greenleaf（导师：[谢晓亮](#谢晓亮)）

主要方向：UNDERSTANDING THE PHYSICAL GENOME；HIGH-THROUGHPUT BIOPHYSICS & MOLECULAR EVOLUTION；DNA ACCESSIBILITY & CHROMATIN STRUCTURE

主要成果：ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide

Homepage: https://greenleaf.stanford.edu/

#### 张元豪 Howard Y. Chang

主要方向：细胞染色质可及性和异质性的研究

主要成果：scATAC-seq

Homepage: https://profiles.stanford.edu/howard-chang

### Cold Spring Harbor Laboratory

#### 张奇伟

主要方向：启动子鉴定；RNA剪切事件检测

主要成果：CEPDB；CSEdb；CIPHER；ESEfinder

Homepage: [Michael Q. Zhang's Lab: People](http://rulai.cshl.edu/people/index1.htm)

### 北京大学

#### 汤富酬

主要方向：人类早期胚胎以及生殖系细胞发育的单细胞功能基因组学研究

主要成果：单细胞转录组测序技术的开发（mRNA-Seq whole-transcriptome analysis of a single cell）

Homepage: [汤富酬-北京未来基因诊断高精尖创新中心](https://icg.pku.edu.cn/kxyj/yjtd/269287.htm)

### 清华大学

#### 颉伟（导师：[任兵](#任兵)）

主要方向：动物胚胎早期发育过程中的表观遗传调控；动物胚胎早期发育过程中的表观遗传调控

主要成果：哺乳动物胚胎发育过程中表观图谱的构建

Homepage: [颉 伟-清华大学生命学院](https://life.tsinghua.edu.cn/info/1034/2439.htm)

### NIH

#### 赵可吉

主要方向：哺乳动物发育和分化的表观遗传机制研究

主要成果：人类组蛋白甲基化图谱构建、人类核小体定位图谱构建

Homepage: https://irp.nih.gov/pi/keji-zhao

### 北京基因组研究所

#### 刘江

主要方向：表观遗传修饰如何从父母遗传到子代的遗传和进化规律进行研究

主要成果：

Homepage: [中国科学院北京基因组研究所（国家生物信息中心）](http://www.big.cas.cn/yjdw_/zgjry/index_84121.html?json=http://www.big.cas.cn/sourcedb_big_cas/zw/zjrc_/zg/201311/t20131116_3979410.json)

### 同济大学

#### 张勇（导师：[刘小乐](#刘小乐)）

主要方向：表观遗传组学算法的开发、组蛋白干细胞分化以及胚胎发育

主要成果：MACS（张勇第一作者，刘小乐通讯作者）

Homepage: https://zhanglab.tongji.edu.cn/index.htm

### 西湖大学

#### 张延晓 （导师：[任兵](#任兵)）

主要方向:  investigating how the dysfunction of heterochromatin, repetitive elements and transposons contributes to the oncogenic and aging processes

主要成果: Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells(**Nature Genetics, 2019**)

Homepage: https://zhangyxlab.github.io/ 

#### 张恺（ 导师：[任兵](#任兵))

主要方向：embracing a multidisciplinary approach to address pressing challenges in Regulatory Genomics

主要成果：A single-cell atlas of chromatin accessibility in the human genome(Cell, 2021)

Homepage: https://lab.kaizhang.org/

###  Northwestern University

#### 岳峰（导师：[任兵](#任兵)）

- 3D genome
- 主要方向：interested in cancer genomics and has demonstrated how epigenome and 3D genome structure are altered and led to gene dysregulation in different types of tumor
- 主要成果： A comparative encyclopedia of DNA elements in the mouse genome(Nature, 2014)
- Homepage: http://yuelab.org/index.html

### Washington University

#### Ting Wang

- 主要方向：investigating the evolutionary model of mobile elements (or transposable elements) and their roles in basic biology and cancer, including their genetic and epigenetic regulation
- 主要成果：Exploring genomic data coupled with 3D chromatin structures using the WashU Epigenome Browser(Nature Methods, 2022)
- Homepage: https://wang.wustl.edu/about/

## 二：基因组学

### Broad Institute of MIT and Harvard

#### Eric S. Lander(谨慎，有负面报道)

主要方向：human genetic variation; human population history; genome evolution; regulatory elements; long non-coding RNAs; three-dimensional folding of the human genome; and genome-wide screens to discover the genes essential for biological processes using CRISPR-based genome editing

主要成果：Broad Institute创始人、人类基因组计划首席科学家、GATK、IGV

Homepage: [Eric S. Lander](https://www.broadinstitute.org/bios/eric-s-lander)

### Genentech

#### Aviv Regev

主要方向：single-cell genomics；The evolution of gene regulation；networks that regulate genes, define cells and tissues, and influence health and disease

主要成果：Fungal orthogroups；Cancer module map；Module networks；international Human Cell Atlas project首席科学家

Homepage: https://www.broadinstitute.org/regev-lab

### Dana-Farber and Harvard Medical School

#### 李恒

主要方向：比对等算法开发

主要成果：SAM格式设计；SAMtools；bwa；Minimap2；Chromap

Homepage: [Heng Li's Homepage](http://www.liheng.org/)

### Massachusetts Institute of Technology

#### Bonnie Berger

主要方向：Computational molecular biology；Network Inference；PPI networks；protein structural motif recognition and discovery；Compressive Genomics；molecular self-assembly and mis-assembly, and functional genomics

主要成果：Global alignment of multiple protein interaction networks with application to functional orthology detection

Homepage: [Bonnie Berger | MIT Mathematics](https://math.mit.edu/directory/profile.php?pid=20)

### Children's Hospital of Philadelphia

#### 王凯

主要方向：bioinformatics methods to advance genomic medicine, deep neural network for long-read sequencing, deep phenotyping on electronic health records, graduate rotation and undergraduate research projects

主要成果：ANNOVAR, wANNOVAR, PennCNV

Homepage: [Wang Lab](https://wglab.org/index.html)

### 北京大学

#### 谢晓亮

主要方向：单细胞全基因组学；单分子酶学；单分子生物物理化学

主要成果：多重退火循环扩增法（MALBAC）

Homepage: [谢晓亮-北京未来基因诊断高精尖创新中心](https://icg.pku.edu.cn/kxyj/yjtd/269284.htm)

#### 张泽民

主要方向：单细胞技术研究肿瘤生物学

主要成果：GEPIA web server、构建多种肿瘤单细胞图谱

Homepage: http://cancer-pku.cn/

#### 李程（导师：[王永雄 Wing Hung Wong](#王永雄 Wing Hung Wong)）

主要方向：肿瘤突变的检测以及基因组突变和基因表达的调控关系

主要成果：Combat；3Disease Browser；突变图谱的构建。

Homepage: [北京大学统计科学中心](https://www.stat-center.pku.edu.cn/zxry/zxjy/lc/1227408.htm)

### 清华大学

#### 张学工 

主要方向：机器学习与生物和医学大数据分析、人类细胞图谱与人体系统数字孪生、单细胞生物信息学分析

主要成果：

Homepage: [张学工-清华大学自动化系](https://www.au.tsinghua.edu.cn/info/1110/1569.htm)

### Cornell University

#### Edward Buckler

主要方向：Plant Breeding and Genetics Section; Quantitative and statistical genetics in maize, cassava, biofuel grasses, and grapes

主要成果：TASSEL: software for association mapping of complex traits in diverse samples(Bioinformatics 2007)

Homepage: https://www.maizegenetics.net/

### 中山大学

#### 吴仲义

主要方向：分子进化、群体基因组学、进化基因组学

主要成果：

Homepage: http://sklbc.sysu.edu.cn/teacher/202

### 西湖大学

#### 杨剑

主要方向：通过GWAS检测基因组变异、高性能计算生物学分析方法和工具的开发、以及疾病的遗传风险评估

主要成果：GCTA、SMR、OSCA、fastGWA以及fastGWA-GLMM

Homepage: https://sls.westlake.edu.cn/Our_Faculty/202007/t20200717_6659.shtml

#### 窦岩梅

主要方向：开发检测一系列常见和罕见嵌合突变的方法; 探索人类不同发育阶段突变谱的特征和之间的潜在关联; 对嵌合突变进行泛癌检测和分析

主要成果：Accurate detection of mosaic variants in sequencing data without matched controls. (**Nat Biotechnol, 2020**)

Homepage: https://sls.westlake.edu.cn/Our_Faculty/202104/t20210407_9156.shtml

### The University of Texas

#### Nicholas Navin

主要方向：Single Cell Genomics; Cancer Evolution；development of single cell genome sequencing technologie；study cancer as an evolutionary process in which clones undergo selection and expansion in response to selective pressures

主要成果：Tumour evolution inferred by single-cell sequencing（Nature 2011）

Homepage: https://faculty.mdanderson.org/profiles/nicholas_navin.html

#### Momiao Xiong

主要方向：Genome-Wide Causation Studies(GWCS)

主要成果：[Genome-Wide Causation Studies of Complex Diseases](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=q1VY28gAAAAJ&sortby=pubdate&citation_for_view=q1VY28gAAAAJ:ubry08Y2EpUC)

Homepage：[google scholar](https://scholar.google.com/citations?hl=en&user=q1VY28gAAAAJ&view_op=list_works&sortby=pubdate)

### University of Michigan

#### 张建之

主要方向：计算分子进化；进化基因组学

主要成果：

Homepage: [Jianzhi Zhang | U-M LSA Ecology and Evolutionary Biology (EEB)](https://lsa.umich.edu/eeb/people/faculty/jianzhi.html)

#### Xiang Zhou

主要方向：Methods for Genome-wide Association Studies，Methods for Functional Genomics Sequencing Studies

主要成果：Genome-wide efficient mixed-model analysis for association studies

Homepage：http://www.xzlab.org/index.html

### University of Chicago

#### 龙漫远

主要方向：Evolution of gene essentiality in development；Evolutionary analysis of gene interactions with new genes

主要成果：

Homepage: [| Ecology & Evolution](https://ecologyandevolution.uchicago.edu/faculty/manyuan-long-phd)

### University College London

#### 杨子恒

主要方向：develop statistical models and computer software for population genetic and phylogenetic

主要成果：BP&P: Bayesian analysis of genomic sequence data under the multispecies coalescent model；Phylogenetic analysis by maximum likelihood （PAML）

Homepage: http://abacus.gene.ucl.ac.uk/

### 复旦大学

#### 金力

主要方向：人类群体遗传学和基因组学、医学遗传学及遗传流行病学（心血管疾病、风湿病、肿瘤等）、计算生物学

主要成果：

Homepage: [金力](https://ibs.fudan.edu.cn/ab/2f/c21163a240431/page.htm)

#### 徐书华

主要方向: 发展和运用群体遗传学方法和计算生物学手段，致力于研究人群遗传结构、环境适应和微观进化等方面的科学问题

主要成果: 

Homepage: http://pog.fudan.edu.cn/#/home

### Carnegie Mellon University

#### 马坚

主要方向：机器学习算法的开发；人类基因组结构和功能与疾病的联系；智能医疗和智能健康；多模态数据整合

主要成果：Infinite Sites Model、InferCARs、Nucleome Browser

Homepage: [Carnegie Mellon School of Computer Science](https://www.cs.cmu.edu/~jianma/)

### 中国农业科学院农业基因组研究所

#### 阮珏   

主要方向：基因组组装算法、极低频点突变检测、基因组结构变异检测

主要成果：Pseudo-sanger、wtdbg2、SMARTdenovo

Homepage: [中国农科院基因组所](https://www.agis.org.cn/kydw/kydwyjzx/zxjsyjzx/b9eb4c62e6d54e12a88f130921563eee.htm)

### St. Jude Children's Research Hospital

#### Jinghui Zhang

主要方向：the development of highly accurate and sensitive computational methods for analyzing large-scale genomic data, especially in the area of detecting and analyzing genetic variations and somatic mutations

主要成果：参与人类基因组计划；开发BLAST（co-author）；CREST maps somatic structural variation in cancer genomes with base-pair resolution（Nature Methods 2011）；Copy Number Segmentation by Regression Tree in Next Generation Sequencing（Nature Methods 2015）

Homepage: https://www.stjuderesearch.org/site/lab/zhang

### 上海交通大学

#### 方海（没有找到主页）

主要方向：GWAS治疗靶点转化研究

主要成果：A genetics-led approach defines the drug target landscape of 30 immune-related traits（Nature Genetics 2019）

Homepage: https://www.nature.com/articles/s41588-019-0456-1#code-availability

### Washington State University

#### Zhiwu Zhang（导师：[Edward Buckler](#Edward Buckler)）

主要方向: 1)high throughput phenotyping by using hyperspectral images and remote sensing images from satellites and drones; 2) the development of innovative statistical methods and computing tools for gene mapping and genomic selection

主要成果: Mixed linear model approach adapted for genome-wide association studies(Nature Genetics, 2010)

Homepage: https://zzlab.net/

### 中国农业科学院农业基因组研究所

#### 黄三文

主要方向：利用组学大数据开拓植物生物学前沿并推动作物育种变革；发起了“优薯计划”，旨在利用基因组设计育种变革马铃薯的育种和繁殖方式

主要成果：Haplotype-resolved genome analyses of a heterozygous diploid potato. (**Nature Genetics, 2020**)

Homepage: https://agis.caas.cn/rcpy/gjjrc/gjgccrctszcjh/230423.htm

### 中科院遗传发育研究所

#### 鲁非（导师：[Edward Buckler](#Edward Buckler)）

主要方向: 

主要成果: witchgrass genomic diversity, ploidy and evolution: novel insights from a network-based SNP discovery protocol(**PLoS Genetics, 2013**)

Homepage: https://plantgeneticslab.github.io/home/

### 华中农业大学

#### 刘小磊（导师：[Edward Buckler](#Edward Buckler)）

主要方向：统计遗传学与基因组育种，致力于全基因组关联分析（GWAS）和基因组选择/预测（GS/GP）算法的研究

主要成果：Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome-wide Association Studies.(**PLoS Genetics, 2016)**

Homepage: https://faculty.hzau.edu.cn/liuxiaolei/zh_CN/index.htm

### 哈尔滨工业大学

#### 王亚东

主要方向: 基因组数据分析算法开发

主要成果: Long-read-based human genomic structural variation detection with cuteSV(**Genome Biology, 2020**)

Homepage: http://homepage.hit.edu.cn/wangyadong

## 三：RNA组学

### 中国科学院大学生物物理所

#### 陈润生（中国生信第一人）

主要方向：胚胎早期发育以及干细胞重编程过程中长非编码RNA以及编码小肽的系统发现和功能机制研究

主要成果：参加人类基因组1%和水稻基因组工作草图的研究；非编码RNA数据库NONCODE；数据库NPInter（非编码RNA与其它生物大分子相互作用）

Homepage: [陈润生-中国科学院大学-UCAS](http://people.ucas.ac.cn/~runshengchen)

### Stanford University

#### 王永雄 Wing Hung Wong

主要方向：贝叶斯统计、计算生物学；基因调控网络

主要成果：采样算法并应用到贝叶斯推理的方法；开发了微阵列基因芯片表达数据和RNA测序数据分析的创新模型和方法

Homepage: [Welcome to the Wong Lab](https://web.stanford.edu/group/wonglab/)

### 北京大学

#### 高歌

主要方向：围绕基因型-表型关系的研究，基于统计建模与机器学习的生物学大数据整合与挖掘、以基因表达为中心的调控通路功能及演化、基于组学大数据的精准医学

主要成果：

Homepage: [Bioinformatics and Computational Genomics](http://www.gao-lab.org/)

### 清华大学

#### 鲁志

主要方向：动植物中编码和非编码RNA的转录后调控研究 、癌症等疾病中新型RNA分子标识物的发现和机理研究、 机器学习等数据挖掘技术的研究及其在基因组学大数据上的应用

主要成果：

Homepage: [鲁 志-清华大学生命学院](https://life.tsinghua.edu.cn/info/1034/1298.htm)

#### 张强锋（导师：[张元豪 Howard Y. Chang](#张元豪 Howard Y. Chang)）

主要方向：使用高通量深度测序的手段来探测RNA二级结构和计算建模，RNA功能模体（motif）等有效预测或发现方法，蛋白-蛋白、RNA-RNA、以及蛋白-RNA相互作用网络

主要成果：VRmol: an Integrative Web-Based Virtual Reality System to Explore Macromolecular Structure （Bioinformatics 2020)；RISE: a database of RNA interactome from sequencing experiments（Nucleic Acids Res 2018)

Homepage: [张强锋实验室](https://zhanglab.net/)

### University of California San Diego

#### Gene Yeo

主要方向：manipulating RNA processing in development and disease using induced pluripotent stem cell and murine models； RNA genomics technology and therapeutics development

主要成果：seCLIP

Homepage: [Institute for Genomic Medicine](http://igm.ucsd.edu/faculty/profiles/gene-yeo.shtml)

### 中科院上海营养与健康研究所

#### 杨力

主要方向：构建高效计算生物学新体系，揭示外显子环形RNA在生成加工和功能作用水平的多层次调控新机制；开展大数据整合及计算生物学分析，主要发现RNA单碱基编辑和修饰互作的分子基础，并进一步利用核酸编辑酶构建高效基因组碱基编辑新体系

主要成果：Circular intronic long noncoding RNAs（Molecular cell 2015）；Complementary sequence-mediated exon circularization（2014 Cell）

Homepage: [杨力----中国科学院上海营养与健康研究所](http://www.sinh.cas.cn/rcdw/qtyjzz/201803/t20180327_4986741.html)

### The University of Texas MD Anderson Cancer Center

#### Han Liang（导师：Wen-Hsiung Li）

主要方向：bioinformatics tool development, integrated cancer genomic analysis, RNA regulation/modification, and cancer systems biology

主要成果：Whole-exome sequencing combined with functional genomics reveals novel candidate driver cancer genes in endometrial cancer；TANRIC: an interactive open platform to explore the function of lncRNAs in cancer

Homepage: https://faculty.mdanderson.org/profiles/liang_han.html

### 浙江大学

#### 郭国骥

主要方向：单细胞RNA组学

主要成果：Construction of a human cell landscape at single-cell level（Cell 2020）;

Mapping the Mouse Cell Atlas by Microwell-Seq(Cell 2018)

Homepage: [郭国骥-浙江大学个人主页](https://person.zju.edu.cn/ggj#0)

### Johns Hopkins University

#### Steven Salzberg

主要方向: develop software for aligning and assembling genomes representing a wide range of species, transcriptome analysis, and microbiome analysis

主要成果: Bowtie2, TopHat2, Cufflinks, HISAT, StringTie and FLASH 

Homepage: https://salzberg-lab.org/

### UCLA

#### Jingyi Jessica Li

主要方向：developing statistical and computational methods，including association measures, high-dimensional variable selection, and classification metrics， scRNA-seq

主要成果：An accurate and robust imputation method scImpute for single-cell RNA-seq data Clipper: p-value-free FDR control on high-throughput data from two conditions

Homepage：http://jsb.ucla.edu/

## 四：蛋白质组学

### Eidgenössische Technische Hochschule Zürich

#### Ruedi Aebersold

主要方向：One of the pioneers in the field of proteomics, Mass Spectrometry, Systems Biology, Bioinformatics, 

主要成果：ICAT & SRM & SWATH

Homepage: [Ruedi Aebersold](http://www.imsb.ethz.ch/research/aebersold/people/ruedi-aebersold.html)


### Max-Planck-Institute of Biochemistry

#### Matthias Mann

主要方向：Mass Spectrometry, Systems Biology, Bioinformatics, Signal Transduction, Posttranslational modifications, Metabolic diseases, Clinical proteomics, Cancer

主要成果：SILAC & MaxQuant

Homepage: [Mann lab](https://www.biochem.mpg.de/mann)


### Scripps Research Institute

#### John R Yates III

主要方向：Mass Spectrometry, Bioinformatics, 

主要成果：SEQUEST & MudPIT

Homepage: [Yates Lab](http://fields.scripps.edu/yates/wp/)

### 西湖大学

#### 郭天南

主要方向：开发前沿蛋白质组学技术，用于高通量地从最少量的临床样本精确定量最多的蛋白质，推动定量蛋白质组学的研究；生物标记的鉴定

主要成果：Rapid mass spectrometric conversion of tissue biopsy samples into permanent quantitative digital proteome maps（2015）；Multi-organ proteomic landscape of COVID-19 autopsies（2020）

Homepage: [Guomics Laboratory](https://guomics.com/)

### 暨南大学

#### 张弓

主要方向：翻译组学

主要成果：FANSe系列超高精度大规模测序序列比对算法；建立了翻译中mRNA与蛋白质丰度之间的定量关系，将中心法则定量化

Homepage: [Translatomics Lab 翻译组学实验室](https://translatome.net/About/Research_interest.html)

## 五、计算结构生物学

### Google DeepMind

#### Demis Hassabis（导师：Tomaso Poggio）

主要方向：neuroscience and artificial intelligence

主要成果：AlphaFold（通讯）

Homepage: [About](https://deepmind.com/about)

#### John Jumper

主要方向：protein structure prediction 

主要成果：AlphaFold（一作）

Homepage: [Science Team](https://deepmind.com/about/science#team_profile)

### The University of Washington

#### David Baker

主要方向：Deep learning for protein structure refinement and protein design; Designing molecular switches, enzymes, and motors; Designing delivery vehicles for targeted intracellular delivery of biologics 

主要成果：Protein Structure Prediction Using Rosetta(Methods in enzymology 2004)

Homepage: https://www.bakerlab.org/

### Tel Aviv University

#### Nir Ben-Tal

主要方向：computational structural biology， including both methods development and applications to selected problems

主要成果：ConSurf、Rate4Site

Homepage: https://en-lifesci.tau.ac.il/profile/bental

### Max Planck Institute for Developmental Biology

#### Andrei N. Lupas

主要方向：Protein Evolution

主要成果：Predicting coiled coils from protein sequences

Homepage: https://www.mpg.de/459944/developmental-biology-lupas

#### Johannes Söding

主要方向：protein function and structure prediction；sequence search and assembly in metagenomics

主要成果：The HHpred interactive server for protein homology detection and structure prediction；

HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment

Homepage: https://www.mpibpc.mpg.de/soeding

### University of Texas Southwestern Medical Center

#### Nick Grishin

主要方向：develop new computational approachesto explore protein sequence-structure

主要成果：PROMALS3D: a tool for multiple protein sequence and structure alignments；AL2CO: calculation of positional conservation in a protein sequence alignment；ECOD；ProSMoS；

Homepage: [Grishin Lab: Home Page](http://prodata.swmed.edu/Lab/HomeLAB.htm)

### University of Michigan Medical School

#### Yang Zhang

主要方向：develop bioinformatics approaches to predict the three-dimensional structures of proteins from amino acid sequences ；Assemble Protein Structures from Cryo-EM

主要成果：I-TASSER（通讯）: a unified platform for automated protein structure and function prediction；TM-align: a protein structure alignment algorithm based on the TM-score；CR-I-TASSER: Assemble Protein Structures from Cryo-EM Density Maps using Deep Convolutional Neural Networks

Homepage: [Yang Zhang, Ph.D. | Computational Medicine and Bioinformatics | Michigan Medicine](https://medicine.umich.edu/dept/dcmb/yang-zhang-phd)

### Memorial Sloan Kettering Cancer Center

#### Dana Pe'er

主要方向：combines single cell technologies, genomic datasets and machine learning algorithms to address fundamental questions in biomedical science

主要成果：Using Bayesian networks to analyze expression data（Journal of computational biology 2000）

Homepage: https://www.mskcc.org/research/ski/labs/dana-pe-er

### 山东大学

#### 杨建益

主要方向：Protein structure and function prediction；Protein structure alignment；Protein-ligand binding site prediciton

主要成果：I-TASSER（一作）

Homepage: [Yang Lab](https://yanglab.nankai.edu.cn/)

### King Abdullah University of Science and Technology

#### 高欣

主要方向：1) developing theory and methodology in the fields of machine learning and algorithms; and 2) solving key open problems in biological and medical fields through building computational models, developing machine-learning techniques, and designing effective and efficient algorithms

主要成果：DEEPre: sequence-based enzyme EC number prediction by deep learning(Bioinformatics 2018)

Homepage: https://sfb.kaust.edu.sa/Pages/Home.aspx

### Chinese University of Hong Kong

#### 李煜（导师：高欣）

主要方向：working at the intersection between machine learning, healthcare and bioinformatics, developing new machine learning methods to resolve the computational problems in biology and healthcare, especially the structured learning problems

主要成果：DEEPre: sequence-based enzyme EC number prediction by deep learning(Bioinformatics 2018)

Homepage: https://liyu95.com/

### 西湖大学

#### Jianyang (Michael) Zeng

主要方向：AI for drug discovery and drug repositioning，Three-dimensional genome structure and gene regulation，Computational method development in Cryo-EM

主要成果：[DTINet](https://github.com/luoyunan/DTINet) , [ROSE](https://github.com/zhangsaithu/rose)

Homepage：https://www.westlake.edu.cn/faculty/jianyang-zeng.html

### 北京生命科学研究所

#### 徐纯福（导师：[David Baker](#David Baker)）

主要方向：开发基于深度学习算法的新的蛋白设计方法

主要成果：Computational Design of Transmembrane Pores.(**Nature, 2020**)

Homepage: http://www.nibs.ac.cn/yjsjyimgshow.php?cid=5&sid=6&id=2467

## 六：计算神经生物学

### The Center for Brains, Minds and Machines (CBMM) of MIT

#### Tomaso Poggio 

主要方向：Brain Imaging；Cellular & Molecular Neuroscience；Cognitive Neuroscience；ComputationalNeuroscience

主要成果：Networks for approximation and learning；Face recognition: Features versus templates；

Hierarchical models of object recognition in cortex

Homepage: https://cbmm.mit.edu/about/people/poggio

### 复旦大学

#### 赵兴明

主要方向：开发新的计算方法用于分子网络的构建和分析；脑核磁共振图像的处理；脑功能神经网络的构建和处理；疾病相关分子通路的识别；药物靶蛋白预测和药物重定位

主要成果：PhosD: inferring kinase–substrate interactions based on protein domains (Bioinformatics 2017)；Prediction of drug combinations by integrating molecular and pharmacological data（PLoS computational biology 2011）

Homepage: http://comp-sysbio.org/

### New York University

#### 汪小京

主要方向： neural computation and memory in cortical circuits

主要成果：[Mechanisms of gamma oscillations](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=cv-YgL0AAAAJ&citation_for_view=cv-YgL0AAAAJ:zYLM7Y9cAGgC)

Homepage: http://www.cns.nyu.edu/wanglab/

## 七：计算生物学(算法开发)

### Stanford AI Lab

#### Daphne Koller

主要方向：Systematic Analysis of Breast Cancer Morphology Uncovers Stromal Features Associated with Survival；Pathways Understanding；Study of Protein-Protein Interactions Using Probabilistic Graphical Models；Computer vision；Bayesian Network；Robotics

主要成果：Activity motifs reveal principles of timing in transcriptional control of the yeast metabolic network；Continuous Time Bayesian Networks；Multi-Class Segmentation with Relative Location Prior

Homepage: [Daphne Koller's Research Group](http://dags.stanford.edu/projects.html)

### Weizmann Institute of Science

#### Nir Freidman

主要方向：combine advanced experimental and mathematical approaches to study intercellular communication；T cell activation and differentiation；Characterizing T cell receptor repertoires using high throughput sequencing (TCR-seq)

主要成果：Gaussian Process Networks

Homepage: https://www.weizmann.ac.il/immunology/NirFriedman/

### 北京大学

#### 韩敬东

主要方向：开发数据整合和网络分析的计算方法；衰老的系统生物学研究

主要成果：Evidence for dynamically organized modularity in the yeast protein–protein interaction network；Spatial transcriptomic analysis of cryosectioned tissue samples with Geo-seq

Homepage: [韩敬东-北京大学前沿交叉学科研究院](http://www.aais.pku.edu.cn/duiwu/showproduct.php?id=214)

### The Chinese University of Hong Kong

#### Kevin Y Yip

主要方向：computational biology and bioinformatics (CBB)；Whole-genome identification of sequence elements；Computational modeling of gene regulation

主要成果：参与ENCODE；Harp: A practical projected clustering algorithm

Homepage: [Home of Kevin Yip](http://www.cse.cuhk.edu.hk/~kevinyip/index.html)

### University of Pennsylvania

#### Nancy Ruonan Zhang

主要方向：Statistics, Computer Science, and Biology, seeking new ways to think about and work with genomic data

主要成果：参与ENCODE；Graph-based change-point detection；A modified Bayes information criterion with applications to the analysis of comparative genomic hybridization data；SAVER: gene expression recovery for single-cell RNA sequencing

Homepage: [Department of Statistics and Data Science](https://statistics.wharton.upenn.edu/profile/nzh/)

#### Kai Tan

主要方向：using genomics and systems biology approaches to understand the gene regulatory factors underlying cellular processes

主要成果：Global view of enhancer–promoter interactome in human cells；A comparative genomics approach to prediction of new members of regulons

Homepage: https://www.med.upenn.edu/apps/faculty/index.php/g275/p8885111

#### Mingyao Li

主要方向：statistical genetics and genomics；use statistical and machine learning approaches to understand cellular heterogeneity ；characterize gene expression diversity across cell types；to study the patterns of cell state transition and crosstalk of various cells using data generated from single-cell transcriptomics studies

主要成果：MetaDiff；PennSeq

Homepage: https://www.med.upenn.edu/apps/faculty/index.php/g275/p8122973

### UT Southwestern Medical Center

#### Jian Zhou

主要方向：Decoding Genomic Sequence；Evolution of Regulatory Genome；Data Science and AI Methods

主要成果：Whole-genome deep-learning analysis identifies contribution of noncoding mutations to autism risk；Predicting effects of noncoding variants with deep learning–based sequence model

Homepage: [Jian Zhou - Home](https://www.princeton.edu/~jzthree/)

### Johns Hopkins University

#### Jean Fan

主要方向：develop methods for analyzing single-cell spatially resolved transcriptomic sequencing and imaging data；cellular heterogeneity on cancer pathogenesis and prognosis

主要成果：VeloViz: RNA-velocity informed embeddings for visualizing cellular trajectories；Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis；Granatum: a graphical single-cell RNA-Seq analysis pipeline for genomics scientists

Homepage: [Jean Fan - Johns Hopkins Biomedical Engineering](https://www.bme.jhu.edu/people/faculty/jean-fan/)

### University of Michigan

#### Lana Garmire

主要方向：Integrative omics/clinic data analysis；Develop computational methods to analyze high-throughput data from next-generation sequencing

主要成果：Cox-nnet: an artificial neural network method for prognosis prediction of high-throughput omics data；Deep learning–based multi-omics integration robustly predicts survival in liver cancer；

Homepage: [Lana Garmire, Ph.D. | Computational Medicine and Bioinformatics | Michigan Medicine](https://medicine.umich.edu/dept/dcmb/lana-garmire-phd)

### Univeristy of Hong Kong

#### Yuanhua Huang

主要方向：developing statistical machine learning methods and computational algorithms for analysing biomedical data, particularly single-cell genomic data

主要成果：BRIE: transcriptome-wide splicing quantification in single cells；Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference

Homepage: [Huang Lab @ HKU Home](https://web.hku.hk/~yuanhua/)
